manifests/init.pp

```diff
@@ -186,6 +186,7 @@
   Array $auth_types                               = $rundeck::params::auth_types,
   Boolean $clustermode_enabled                    = $rundeck::params::clustermode_enabled,
   Hash $database_config                           = $rundeck::params::database_config,
+  $execution_mode                                 = $rundeck::params::execution_mode,
```

Comment:
@juniorsysadmin good point.
I think `Enum['active', 'passive'] $execution_mode =  ...`

Reply:
Another option would be either `Optional[Enum['active', 'passive']] $execution_mode = undef,` and then suppress it unless set explicitly. Something like `Bool $enable_passive_execution` [defaulting to false] could also theoretically work.